packages/shared-metrics/package-lock.json

```diff
@@ -30,6 +30,29 @@
         "js-tokens": "^4.0.0"
       }
     },
+    "@instana/core": {
+      "version": "1.106.2",
+      "resolved": "https://registry.npmjs.org/@instana/core/-/core-1.106.2.tgz",
+      "integrity": "sha512-ROqPpighGgLkR/g7+/qprdDa9ahP2ZnKAi8DrvvBV34/uL0nL3c701K5ZCXKZIJW9t/Fdgx3OBj1rf+RHmrNfg==",
+      "requires": {
+        "async-hook-jl": "^1.7.6",
+        "cls-bluebird": "^2.1.0",
+        "emitter-listener": "^1.1.2",
+        "lru-cache": "^5.1.1",
+        "methods": "^1.1.2",
+        "opentracing": "^0.14.3",
+        "redis-commands": "^1.3.5",
+        "semver": "5.5.1",
+        "shimmer": "1.2.0"
+      },
+      "dependencies": {
+        "semver": {
+          "version": "5.5.1",
+          "resolved": "https://registry.npmjs.org/semver/-/semver-5.5.1.tgz",
+          "integrity": "sha512-PqpAxfrEhlSUWge8dwIp4tZnQ25DIOthpiaHNIthsjEFQD6EvqUKUDM7L8O2rShkFccYo1VjJR0coWfNkCubRw=="
+        }
+      }
+    },
```

Comment:
Running `npm install` in `packages/shared-metrics` will add this (and all of `@instana/core`'s transitive dependencies, but due to the way this monorepo is managed with lerna, this is actually not supposed to be here, even though `@instana/core` is a dependency of `@instana/shared-metrics`. I'll take care of cleaning that up.